plugins.vim

```diff
@@ -109,15 +91,19 @@ Plug 'avakhov/vim-yaml'                              " Yaml stuff
 
 """" JAVASCRIPT
 """"" BASE
-Plug 'othree/yajs.vim'
-Plug 'othree/javascript-libraries-syntax.vim'
-Plug 'othree/es.next.syntax.vim'
-Plug 'gavocanov/vim-js-indent'
-Plug 'isRuslan/vim-es6'                              " es-6 help
+" Plug 'othree/yajs.vim'
+" Plug 'othree/javascript-libraries-syntax.vim'
+" Plug 'othree/es.next.syntax.vim'
+" Plug 'gavocanov/vim-js-indent'
+" Plug 'isRuslan/vim-es6'                              " es-6 help
+" Plug 'pangloss/vim-javascript'
```

Comment:
The typescript plugins and language server seem to take care of all of these, but i left them commented in case i missed something

Reply:
Interesting. Does this prevent things from working in a non TS project?